.github/pull-request-template.md

```diff
@@ -49,13 +49,13 @@ Before you submit this PR, please make sure:
 - [ ] You have thoroughly tested your changes.
 - [ ] You have followed all the contributing guidelines for this project.
 - [ ] You understand and agree that your contributions will be publicly available 
-  under the project’s license.
+  under the project's license.
```

Comment:
I'm replacing typographic smart quotes with its dumb equivalents

Instead of `“` or `”`, it is `"`, for example.

Why?

* CLIs and source control systems play better with dumb quotes
* automation tools play better with dumb quotes
* Clean is better than clever.
* The only place typography matters is the HTML documentation, and Jekyll already converts dumb quotes to smart quotes there.
* But overall, as a typography enthusiast, I have given up on using smart quotes: All of the tooling I use (including Grammarly, text editors, LLMs, GitHub CoPilot, email assistants, etc, etc use dumb quotes (because they are mostly developed by programmers who cannot care less; and also because it is harder to maintain proper typographic context programmatically (so using dumb quotes simplifies the problem domain so that the programmers will focus on other important stuff)

Anyhoo, from now on it’s simplicity FTW.

Reply:
Interesting, never thought about it.